.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Generated txt from csv files for task 1.2
+task1/csv/*.txt
```

Comment:
write stream rewrite file with the same name
So, if you will have output with static file name. this code become unnecessary

Reply:
Anyway txt file should not appear in repo so `*.txt` works as expected and doesn't care about actual file name